Overview.src.html

```diff
@@ -1387,6 +1414,15 @@ <h3 id="should-response-to-request-be-blocked-due-to-mime-type?"><dfn title="sho
 <span data-anolis-ref>REFERRER</span>
 
 
+<h3>Client hints list</h3>
+
+<p>A <dfn title=concept-client-hints-list>client hints list</dfn> is a list of
+<a href=http://httpwg.org/http-extensions/client-hints.html#accept-ch>Client hint tokens</a>,
+each of which is one of `<code title>dpr</code>`,
+`<code title>save-data</code>`, `<code title>viewport-width</code>`, or
+`<code title>width</code>`.
```

Comment:
It might make sense to define this concept ("client hints list") directly in CLIENT-HINTS too, especially if it is going to be reused by multiple concepts.